src/tools/importPlanGoals.test.js

```diff
@@ -21,7 +21,7 @@ describe('Import TTA plan goals', () => {
         const fileName = 'GranteeTTAPlanTest.csv';
         downloadFile.mockResolvedValue({ Body: readFileSync(fileName) });
         await Role.destroy({ where: {} });
-        await Topic.destroy({ where: {} });
+        await Topic.destroy({ where: {}, force: true });
         await Goal.destroy({ where: {} });
```

Comment:
Apparently findOrCreate fails if a topic has been soft deleted (i.e. the model has a "paranoide: true" flag on it)